openslides/motions/static/js/motions/site.js

```diff
@@ -1305,10 +1305,14 @@ angular.module('OpenSlidesApp.motions.site', [
             return Boolean(isAllowed);
         };
 
+
         // Inline editing functions
         $scope.inlineEditing = MotionInlineEditing;
         $scope.inlineEditing.init($scope, motion);
 
+        // TODO: properties of inlineEditing are currently not watched continuosly,
```

Comment:
The inlineEditing.editor.onChange event modifies inlineEditing.changed, but this is only registered by the 'ng-if' triggers in the template when the object is rerendered (by the tab losing focus). 

I think it is something in angularjs. The result is the "save button" not appearing on time.